src/widgets/MeterPanel.cpp

```diff
@@ -1126,8 +1126,8 @@ void MeterPanel::OnMeterUpdate(wxTimerEvent & WXUNUSED(event))
 
    if (numChanges > 0) {
       #ifdef EXPERIMENTAL_AUTOMATED_INPUT_LEVEL_ADJUSTMENT
-         if (gAudioIO->AILAIsActive() && mIsInput && !discarded) {
-            gAudioIO->AILAProcess(maxPeak);
+         if (AudioIO::Get()->AILAIsActive() && mIsInput && !discarded) {
+            AudioIO::Get()->AILAProcess(maxPeak);
             putchar('\n');
```

Comment:
But this code is disturbing. Why is it here? 

This transformation is not quite conservative anyway, so probably this line should be removed?

It seems that AILA logs extensively, probably way to extensively. Is it ready for use at all?